knsk.sh

```diff
@@ -201,7 +201,8 @@
           pp t4n $RES 
           if (( $DELRES )); then
             CMD1="timeout $TIME $K -n $NS --grace-period=0 --force=true delete $RES"
-            CMD2="timeout $TIME $K -n $NS patch $RES -p '{\"metadata\":{\"finalizers\":null}}'"
+            CMD2="timeout $TIME $K -n $NS patch $RES --type json \
```

Comment:
We need to check the compatibility of this command with old kubenertes versions... do you know in which version... let me spin up some clusters and test it o check. What is your Kubernetes version? Thanks.

Reply:
It work on 1.19 - 1.20 - 1.21 
Last test was on 1.19